### PR TITLE
style: `jest/expect-expect` rule

### PR DESCRIPTION
**Pull request by tuliomir**
### Acceptance Criteria
- Fixes the [`expect-expect` rule](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/expect-expect.md)

### Future possibilities
We could refactor all the assertion helper functions to have the same prefix or suffix, so that we could replace the many exact names in `eslintrc.js` with a single entry that would cover all of them.

It would also be more straightforward to add new helper functions using the same pattern instead of adding them to the `eslintrc.js`.

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by alexruzenhack — APPROVED** [review]

File: __tests__/websocket/websocket.test.ts [hunk 1]
```diff
@@ -46,9 +46,11 @@ describe('onMessage', () => {
 
   it('should receive messages after initializing the timeoutTimer', () => {
     const wsInstance = new GenericWebSocket(baseWsOptions);
+    const msgSpy = jest.spyOn(wsInstance, 'onMessage').mockImplementation(jest.fn());
 
     wsInstance.sendPing();
     wsInstance.onMessage({ data: JSON.stringify({ type: 'other' }) });
+    expect(msgSpy).toHaveBeenCalled();
```
Comment (andreabadesso) [inline __tests__/websocket/websocket.test.ts:53]: So this test wasn't doing anything?

**Review by andreabadesso — APPROVED** [review]

File: __tests__/integration/nanocontracts/bet.test.ts [hunk 2]
```diff
@@ -312,6 +312,7 @@ describe('full cycle of bet nano contract', () => {
     expect(txHistory).toHaveLength(4);
   };
 
+  /* eslint-disable jest/expect-expect -- All assertions are on the helper function */
```
Comment (andreabadesso) [inline __tests__/integration/nanocontracts/bet.test.ts:315]: _Comment unrelated to line number_

You can accomplish the same thing, without disabling the rule by adding `expect.hasAssertions` in `beforeEach` of a shared jest file and adding `additionalTestBlockFunctions` as a parameter to the `jest/expect-expect` rule in the config, e.g.:

```json
{
  "rules": {
    "jest/expect-expect": [
      "error",
      { "additionalTestBlockFunctions": ["executeTests", "testLockedUtxoMethods"] }
    ]
  }
}
```

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: __tests__/websocket/websocket.test.ts (hunk 1 above)
Comment (tuliomir) [reply __tests__/websocket/websocket.test.ts:53]: That's correct. It would throw if the code failed, so it increased our coverage, but had no assertions.

File: __tests__/integration/nanocontracts/bet.test.ts (hunk 2 above)
Comment (tuliomir) [reply __tests__/integration/nanocontracts/bet.test.ts]: The `additionalTestBlockFunctions` didn't work, but I managed to make the same improvement using `assertFunctionNames`.

Do you agree with this approach?

**Review by andreabadesso — APPROVED** [review]

**Review by alexruzenhack — APPROVED** [review]
